app/docker/views/swarm/visualizer/swarmVisualizerController.js

```diff
@@ -1,17 +1,25 @@
 angular.module('portainer.docker')
-.controller('SwarmVisualizerController', ['$q', '$scope', '$document', '$interval', 'NodeService', 'ServiceService', 'TaskService', 'Notifications',
-function ($q, $scope, $document, $interval, NodeService, ServiceService, TaskService, Notifications) {
+.controller('SwarmVisualizerController', ['$q', '$scope', '$document', '$interval', 'NodeService', 'ServiceService', 'TaskService', 'Notifications', 'LocalStorage',
+function ($q, $scope, $document, $interval, NodeService, ServiceService, TaskService, Notifications, LocalStorage) {
 
   $scope.state = {
-    ShowInformationPanel: true,
-    DisplayOnlyRunningTasks: false,
-    refreshRate: '5'
+    ShowInformationPanel: LocalStorage.getSwarmVisualizerSettings('show_info_panel', true),
```

Comment:
Default values should be defined here. Update from local storage in the `initView()` function.